flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkManifestUtil.java

```diff
@@ -61,10 +61,11 @@ static ManifestFile writeDataFiles(OutputFile outputFile, PartitionSpec spec, Li
     }
   }
 
-  static ManifestOutputFileFactory createOutputFileFactory(Table table, String flinkJobId, int subTaskId,
-                                                           long attemptNumber) {
+  static ManifestOutputFileFactory createOutputFileFactory(Table table, String flinkJobId, String operatorUniqueId,
+      int subTaskId, long attemptNumber) {
```

Comment:
nit: this indention seems not matching the current code style

Reply:
Done, but I found the ci had passed. Never mind, I have fixed it.